.travis.yml

```diff
@@ -58,5 +58,7 @@ script:
   # in the makefile
   - if [ $REPO_PROJ == "fvp" ]; then mkdir -p $HOME/$REPO_PROJ/Foundation_Platformpkg; fi
   - cd $HOME/$REPO_PROJ && repo init -u https://github.com/OP-TEE/manifest.git -m $REPO_PROJ.xml </dev/null && repo sync -j2 --no-clone-bundle --no-tags --quiet
+  # Fetch a local copy of dtc+libfdt to avoid issues with a possibly outdated libfdt-dev
+  - if [ $REPO_PROJ == "qemu_v8" ]; then cd $HOME/$REPO_PROJ/qemu && git submodule update --init dtc; fi
```

Comment:
I disagree, because:
1. All submodules in qemu/ are at the top-level, in other words: `--recursive` would be a no-op
2. More importantly, fetching more submodules than needed just brings more opportunities for failures because not all repositories are on the same server. And it's quite infuriating to see a build fail due to something that's not even needed ;)